packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -214,12 +214,231 @@ export class OdspDocumentStorageService extends OdspDocumentStorageServiceBase {
 	}
 
 	public async getSnapshot(snapshotFetchOptions?: ISnapshotFetchOptions): Promise<ISnapshot> {
-		assert(this.hostPolicy.supportGetSnapshotApi !== true, "api should not be called yet");
-		// This is just temporary as this api is not yet enabled in service policies.
-		return this.fetchSnapshot(
-			this.hostPolicy.snapshotOptions,
-			snapshotFetchOptions?.scenarioName,
+		// Fetch full snapshot in case group is not specified in request.
+		let fetchFullSnapshotContents = true;
```

Comment:
I think we should not use "full snapshot" terminology here, as "full snapshot" is more what legacy clients expect and get. Maybe snapshot with default groupId?

Reply:
I think what it really means is fetchFirstSnapshot or fetchSnapshotForLoad. By "fetchFullSnapshotContents" just meant that the first snapshot is full in sense if there are no specific groupids. Can try to rename it

Reply:
Removed this.